repose-aggregator/components/filters/keystone-v2/src/main/resources/META-INF/schema/config/keystone-v2.xsd

```diff
@@ -108,6 +108,13 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="set-rackers-groups" type="xs:string" use="optional">
```

Comment:
I don't love this being another attribute side-by-side with the "set-groups-in-header" attribute. Also, I'm not sure about the name.